.github/policies/labelAdded.highestVersionRemaining.yml

```diff
@@ -1,6 +1,6 @@
-id: labelAdded.lastVersionRemaining
+id: labelAdded.highestVersionRemaining
 name: GitOps.PullRequestIssueManagement
-description: Handlers when "Last-Version-Remaining" label is added
+description: Handlers when "Highest-Version-Remaining" label is added
```

Comment:
If I'm being nitpicky, I think "Remaining" word doesn't make sense anymore (if I'm understanding how you're action works correctly). We can go for something like "Removing-Highest-Version" (or if someone got a better one, naming is hard)

Reply:
I'm hoping `Removal` might be more clear.